src/Symfony/Component/Finder/Tests/FinderTest.php

```diff
@@ -1647,7 +1764,7 @@ public function testIgnoredAccessDeniedException()
                 'qux_10_2.php',
                 'qux_12_0.php',
                 'qux_2_0.php',
-                ]
+            ]
```

Comment:
The code format checker wanted me to unindent this `]`, even though it is indented in the current 7.1 branch code.
I suppose that the current code in the 7.1 branch does not pass the code format checker - strange.